tests/atomistics/structure/test_pyscal.py

```diff
@@ -92,7 +92,11 @@ def test_analyse_pyscal_centro_symmetry(self):
         self.assertTrue(all([np.isclose(v, 0.0) for v in self.al_fcc.analyse_pyscal_centro_symmetry()]))
         self.assertTrue(all([np.isclose(v, 6.177675) for v in self.fe_bcc.analyse_pyscal_centro_symmetry()]))
         self.assertTrue(all([np.isclose(v, 8.7025) for v in self.ti_hcp.analyse_pyscal_centro_symmetry()]))
-        self.assertTrue(all([np.isclose(v, 22.113675) for v in self.si_dia.analyse_pyscal_centro_symmetry()]))
+        print(self.si_dia.analyse_pyscal_centro_symmetry())
+        self.assertTrue(all([
```

Comment:
I found this hard to read. So far I've got this as an improvement:
```python
self.asserTrue(all([
    any(np.isclose(centrosym, [22.113675, 16.58525625, 11.0568375])) 
    for centrosym in self.si_dia.analyse_pyscal_centro_symmetry()
])
```

But honestly that's not yet much improvement.

Reply:
```suggestion
        self.assertTrue(np.allclose
            np.unique(self.si_dia.analyse_pyscal_centro_symmetry()), 
            [11.0568375, 16.58525625, 22.113675]
        ))
```

Also works, but I'm actually more comfortable with forcing the sort.

I find these easier to understand the intent, but it might just be perspective

Reply:
Also, I'm new to these suggestions and I don't see how to recommend getting rid of the lines below this as well...

Reply:
> Also, I'm new to these suggestions and I don't see how to recommend getting rid of the lines below this as well...

You click the plus and then move your cursor down